packages/components/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -89,15 +90,10 @@ export const SegmentedControl: FC<SegmentedControlProps> = ({
       total: labels.length,
     })
 
-    // TODO: Replace with typography tokens
-    const font: TextStyle = {
-      fontFamily: isSelected ? 'SourceSansPro-Bold' : 'SourceSansPro-Regular',
-      fontSize: 20,
-      lineHeight: 30,
-    }
-
     const textStyle: TextStyle = {
-      ...font,
+      ...typography.vadsFontBodyLarge,
+      fontFamily: isSelected ? 'SourceSansPro-Bold' : 'SourceSansPro-Regular',
+      marginBottom: 0,
       color: theme.vadsColorForegroundDefault,
```

Comment:
Did none of the tones work here? Seems like with the Text updates to include `bold` and `center` that maybe this could still use the Text component?

Reply:
Couldn't use Text component here due to the `allowFontScaling` prop used on line 121, unless we add that prop.